migrations/000001_create_users_table.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE public.users
+(
```

Comment:
We are currently using Snowflake ids which produce numeric strings of fixed length. Could a fixed length varchar field be more effective here?

Reply:
I've read, that an extra cycle is executed by query to check the length of stored `varchar(n)` value, so if we don't want an extra constraint, we should probably leave it like that.

Reply:
This is strange. I always thought that index scan should have everything it needs in one place, and no extra roundtrips are required. But this does not matter much right now. Let's keep it the way it is for the time being and do some extra research when we have time.